lib/_debugger.js

```diff
@@ -1557,7 +1556,16 @@ Interface.prototype.repl = function() {
 
     // Exit debug repl
     self.exitRepl();
-  });
+    
+    self.repl.rli.removeListener('SIGINT', exitDebugRepl);
+    self.repl.removeListener('exit', exitDebugRepl);
+  }
+  
+  // Exit debug repl on Ctrl + C
+  this.repl.rli.on('SIGINT', exitDebugRepl);
+  
+  // Exit debug repl on '.exit'
```

Comment:
These comments aren't so correct imo. `ctrl+c` isn't the only way to send a `SIGINT` to a procees, and `exit` can also be caused by `ctrl+d` on an empty repl prompt.